src/garage/torch/algos/vpg.py

```diff
@@ -389,9 +390,9 @@ def _get_baselines(self, path):
                 where T is the path length experienced by the agent.
 
         """
-        if hasattr(self.baseline, 'predict_n'):
-            return torch.Tensor(self.baseline.predict_n(path))
```

Comment:
I see that this particular line is not covered in tests. The use of probing `hasattr` is ugly. We need a unified baseline interface

Reply:
yeah, I will clean this part when I make new value function class for PyTorch.